reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -6033,6 +6033,7 @@ public int getPrefetch() {
 
 	/**
 	 * Emit only the first item emitted by this {@link Flux}, into a new {@link Mono}.
+	 * If called on empty {@link Flux}, emit an empty {@link Mono}.
```

Comment:
improving grammar/sentence: "on `an` empty Flux" and "emit`s` an empty Mono"

Reply:
My bad. Thank you for the review! Fixed